### PR TITLE
oo_collect: Allow list elements to be lists of dict

### PR DESCRIPTION
**Pull request by jarrpa**
This allows for gathering things like the `Ready` status of multiple pods.

Also tweaking the failure comments to better reflect where the messages are coming from. :)

### CODE REVIEW

File: filter_plugins/oo_filters.py [hunk 1]
```diff
@@ -138,21 +139,44 @@ def oo_collect(data, attribute=None, filters=None):
             attribute = 'a'
             filters   = {'z': 'z'}
             returns [1, 2, 3]
+
+        This also deals with lists of lists with dict as elements.
+        Ex: data = [
```
Comment (abutcher) [inline filter_plugins/oo_filters.py]: s/data/data_list/

Comment (jarrpa) [inline filter_plugins/oo_filters.py]: Updated.

**Comment by abutcher** [comment]
aos-ci-test

**Comment by abutcher** [comment]
[merge]

**Pull request merged**